services/auth/test/client_test.js

```diff
@@ -1,9 +1,10 @@
-const assert = require('assert');
+const assert = require('assert').strict;
```

Comment:
Oh, can we just make this import `strict` everywhere rather than updating all calls?

Reply:
That's my understanding from the docs anyway, yeah.